### DISCUSSION TITLE
Unable to Install - Apple M1 Pro

### DISCUSSION THREAD

**Discussion by Dom-pseudo-engineer** [openai/whisper#1016] [Q&A]
![Screen Shot 2023-03-02 at 9 42 03 AM](https://user-images.githubusercontent.com/108111306/222508923-0352cee4-6e4e-4ab6-a27f-4d460dd0e6db.png)
Looks like it has something to do with python3.11, python2.7, and python3.10. Any suggestions?

**Comment by Dom-pseudo-engineer** [comment]
I can't facepalm hard enough.
`pip3.10 install -U openai-whisper`

**Reply by PacoH** [reply]
Did this work?

**Comment by PacoH** [comment]
Having had many problems installing GitHub projects on my iMac the '*nix' way, I now try to find a `brew` formula first. Fortunately there is one for `whisper`. I can't imagine how many problems I'd have installing it with `pip`. As it is, I had 3 errors installing it with `brew`, but I just followed the suggestions given and that solved the problems. It installed many dependencies and updated installed ones.

	brew install openai-whisper

	==> Fetching dependencies for openai-whisper: jpeg-turbo, zstd, jpeg-xl, aom, dav1d, freetype, openssl@1.1, libnghttp2, gnutls, glib, libxcb, libx11, harfbuzz, libass, libvpx, libsndfile, sdl2, libarchive, pango, ffmpeg, git-lfs, python-typing-extensions, sqlite, python@3.11, libyaml, pyyaml, huggingface-cli, numpy, eigen, protobuf, pybind11 and pytorch

**Comment by edrozenberg** [comment]
Thanks to https://github.com/openai/whisper/pull/1171#issuecomment-1509551063

Was able to install whisper on MB Air M2:
```
pip3.11 install -U "numba==0.57.0rc1"
pip3.11 install openai-whisper
```

Did a quick test with the whisper "tiny" model, appears to work fine.
